src/util/color/colorpalette.h

```diff
@@ -6,10 +6,13 @@
 
```

Comment:
Add documentation:
"An ordered list of colors that can be picked by WColorPicker, used for cue and track colors. Also used to map default colors to hotcues based on their hotcue number"